skimage/draw/draw3d.py

```diff
@@ -63,6 +63,112 @@ def ellipsoid(a, b, c, spacing=(1., 1., 1.), levelset=False):
     return arr
 
 
+def ellipsoid_coords(d, r, c, d_radius, r_radius, c_radius, shape=None,
+                     rot_z=0., rot_y=0., rot_x=0., spacing=(1., 1., 1.)):
```

Comment:
Thanks for the contribution @ksugar! I'm a fan of your work. :blush: One thing that we're trying to do in skimage more generally is make our API n-dimensional-friendly, even if the underlying function is not *yet* n-dimensional. So, in this case, I would suggest:

```suggestion
def ellipsoid_coords(center, axis_lengths, *, shape=None,
                     rotation=(0., 0., 0.), spacing=(1., 1., 1.)):
```

I would then add a note in the docstring that the rotation is in ZYX Euler angles, and that this only supports 3D for now.

Also, I have euler matrix creation implemented in #5188, maybe there's the opportunity to build this on top of that one... I'll try to finish that off! btw if you have a bit of experience with 3D geometry/transforms I would totally appreciate a review there! :sweat_smile: